examples/pylab_examples/boxplot_demo2.py

```diff
@@ -16,6 +16,7 @@
 randomDists = ['Normal(1,1)', ' Lognormal(1,1)', 'Exp(1)', 'Gumbel(6,4)',
                'Triangular(2,9,11)']
 N = 500
+np.random.seed(0)
```

Comment:
There are many issues with the examples so I wouldn't expect to fix all of them but I would like to do this in new examples